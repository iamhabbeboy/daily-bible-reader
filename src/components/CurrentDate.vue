<template>
  <div>
    <h1 class="text-xxl text-gray-50 ">
      {{ getTime }}
    </h1>
    <h4 class="text-gray-100">Thur Feb. 18th</h4>
  </div>
</template>

<script>
export default {
  computed: {
    getTime() {
      const time = new Date();
      return `${time.getHours()}:${time.getMinutes()}`;
    }
  }
};
</script>
