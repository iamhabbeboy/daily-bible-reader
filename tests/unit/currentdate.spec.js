import { shallowMount } from "@vue/test-utils";
import CurrentDate from "@/components/CurrentDate";

describe("CurrentDate.vue", () => {
  it("take snapshot", () => {
    const wrapper = shallowMount(CurrentDate);
    expect(wrapper).toMatchSnapshot();
  });
});
