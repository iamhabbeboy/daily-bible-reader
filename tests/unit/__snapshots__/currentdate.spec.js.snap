// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentDate.vue take snapshot 1`] = `
<div>
  <h1 class="text-xxl text-gray-50 ">
    14:36
  </h1>
  <h4 class="text-gray-100">Thur Feb. 18th</h4>
</div>
`;
